content/docs/1.7.0/v2-data-engine/prerequisites.md

```diff
@@ -21,6 +21,11 @@ Longhorn nodes must meet the following requirements:
   >
   > Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues.
 
+  6.7 or later is recommended for an improved system stability
+  > **NOTICE**
+  >
+  > Host machines with Linux kernel prior to 6.7 can encounter memory corruption when IO timeouts are massively present in communications between nvme-tcp driver and SPDK. Update the Linux kernel on Longhorn nodes to version 6.7 for a better error handling to prevent such issue.
```

Comment:
```suggestion
v6.7 or later is recommended for improved system stability
> **NOTICE**
>
> Memory corruption may occur on hosts using versions of the Linux kernel earlier than 6.7. This is caused by prevalent IO timeouts in communications between the nvme-tcp driver and SPDK. Update the Linux kernel on Longhorn nodes to version 6.7 or later to prevent the issue from occurring.
```

Reply:
Just a little change to your suggestion.